website/pages/en/index.js

```diff
@@ -100,8 +95,8 @@ class Index extends React.Component {
 								<section className='icon-wrap'>
 									<div className='icon smart-contract' />
 								</section>
-								<h4>Smart Contract Development</h4>
-								<p>Learn how to build Wasm smart contracts with ink!.</p>
+								<h4>Smart Contracts</h4>
+								<p>Use the WebAssembly based ink! language or EVM-compatible contracts.</p>
```

Comment:
This is awkward because "WebAssembly based ink! language" and "EVM-compatible contracts" are not semantically equivalent. Something like:

"Use the ink! language to build Wasm-based contracts, or deploy Solidity code to an EVM on Substrate."

Reply:
I want to keep these descriptions to two lines for visual reasons so I used something slightly different.